packages/xo-server-perf-alert/src/templates/index.js

```diff
@@ -0,0 +1,83 @@
+const Handlebars = require('handlebars')
+const { join, parse } = require('node:path')
+const { readdirSync, statSync, readFileSync } = require('node:fs')
+
+function humanReadableRule(rule) {
+  const VARIABLE_TO_TEXT = {
+    cpuUsage: 'Cpu usage',
+    memoryUsage: 'Memory usage',
+    storageUsage: 'Storage usage ',
+  }
+  const TYPE_TO_TEXT = {
+    host: 'host',
+    SR: 'storage repository',
+    VM: 'virtual machine',
+  }
+  const COMPARATOR_TO_TEXT = {
+    '<': 'less than',
+    '>': 'more than',
```

Comment:
to avoid `less than than ${rule.triggerLevel}`
```suggestion
    '<': 'less',
    '>': 'more',
```